jest.config.js

```diff
@@ -54,7 +47,10 @@ module.exports = {
 		'\\.(scss)$': '<rootDir>/' + configPath + '/__mocks__/styleMock.js',
 		'\\.(jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
 			'<rootDir>/' + configPath + '/__mocks__/fileMock.js',
-		...transpiledPackages,
+		[ `@wordpress\\/(${ transpiledPackageNames.join(
+			'|'
+		) })$` ]: '<rootDir>/gutenberg/packages/$1/src',
+		'test/helpers$': '<rootDir>/' + configPath + '/helpers.js',
```

Comment:
The logic for mapping the WordPress modules has been updated based on the Jest configuration we have in Gutenberg for unit tests:
- https://github.com/WordPress/gutenberg/blob/43d950788629daaf588c80a2a56173124f112c60/test/native/jest.config.js#L18-L20
- https://github.com/WordPress/gutenberg/blob/43d950788629daaf588c80a2a56173124f112c60/test/native/jest.config.js#L55-L58